app/controllers/invites_controller.rb

```diff
@@ -0,0 +1,52 @@
+require_relative "../views/invites/render_json"
+
+class InvitesController < ApplicationController
+  include InvitesJson
+
+  def index
+    user = @current_user
+    render status: 200,
+      json: groups_invited_to(user)
+  end
+
+  # POST request for inviting another user to a group.
+  def create
```

Comment:
This is the `create` action, so you only give the detail for one invite/membership, the one you want to create :)